common/src/main/java/io/netty/util/internal/ClassInitializerUtil.java

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright 2021 The Netty Project
+ *
+ * The Netty Project licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+package io.netty.util.internal;
+
+/**
+ * Utility which ensures that classes are loaded by the {@link ClassLoader}.
+ */
+public final class ClassInitializerUtil {
+
+    private ClassInitializerUtil() { }
+
+    /**
+     * Preload the given classes and so ensure the {@link ClassLoader} has these loaded after this method call.
+     *
+     * @param classLoader   the {@link ClassLoader}
+     * @param classes    the classes to load.
+     */
+    public static void tryLoadClasses(ClassLoader classLoader, Class<?>... classes) {
```

Comment:
nit: could simplify the calling code by taking the "native" class as the first arg instead of a `ClassLoader`, since there is always such a class the way that this is used (call `PlatformDependent.getClassLoader(...)` here)

Reply:
done :)